dts/bindings/clock/st,stm32f0-rcc.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32F0 and G0 Reset and Clock controller node.
+  For more description confere st,stm32-rcc.yaml
+
+compatible: "st,stm32f0-rcc"
```

Comment:
F0 and G0 use the same hardware block (at least from what is used in zephyr today).
"st,stm32f0-rcc" is provided as the description of this block and then should be used for both F0 and G0 targets.

This field is used for the binding of the compatible with the binding. I don't think it's possible to set 2 bindings here (but indeed this would be handy)